Src/AutoFixtureUnitTest/RegularExpressionGeneratorTest.cs

```diff
@@ -103,6 +103,31 @@ public void CreateWithRegularExpressionRequestReturnsCorrectResult(string patter
             Assert.True(Regex.IsMatch(result.ToString(), pattern), string.Format("result: {0}", result));
         }
 
+        [Theory]
+        [InlineData(@"^[A-Z]{27}$")]
+        public void CreateMultipleWithRegularExpressionRequestReturnsDifferentResults(string pattern)
+        {
+            // This test exposes an issue with Xeger/Random.
+            // Xeger(pattern) internally creates an instance of Random with the default seed.
+            // This means that the RegularExpressionGenerator might create identical strings
+            // if called multiple times within a short time.
+
+            // Arrange
+            var sut = new RegularExpressionGenerator();
+            var request = new RegularExpressionRequest(pattern);
+            var dummyContext = new DelegatingSpecimenContext();
+
+            // Repeat a few times - because the issue seen comes from Random class using tickcount as seed.
+            for (int i = 0; i < 10; i++)
+            {
+                var result1 = sut.Create(request, dummyContext);
+                var result2 = sut.Create(request, dummyContext);
+
+                // Assert
+                Assert.NotEqual(result1, result2);
```

Comment:
Wouldn't it be possible to avoid the `for-loop` and the [Assertion Roulette](http://xunitpatterns.com/Assertion%20Roulette.html) by doing something similar to what you did [here](https://github.com/moodmosaic/Fare/pull/56/commits/1563f8e5cea71bb58c85e7843498796d81975715#diff-a4691edefa1cec0b62875a993b25ed7c)? 👀

Reply:
Yes - totally, just didn't know if introducing linq in this test was acceptable? Commit on the way.